### DISCUSSION TITLE
Neo4j to FlakorDB migration documentation

### DISCUSSION THREAD

**Discussion by JPonsa** [FalkorDB/FalkorDB#659] [Q&A]
I have a KG in Neo4j. I would like to test FalkoDB. Is there a way to migrate by Neo4j db to FalkorDB. Maybe through loading a dump file?

**Comment by gkorland** [comment]
You should check the [falkordb-bulk-loader](https://github.com/FalkorDB/falkordb-bulk-loader).
It allows you to do bulk loading of CVS files.
